nucleus/parent/pom.xml

```diff
@@ -928,11 +930,7 @@
                         <configuration>
                             <resources>
                                 <resource>
-                                    <directory>${maven.multiModuleProjectDirectory}</directory>
```

Comment:
It worked, because the maven.multiModuleProjectDirectory pointed to the root of the project. But it was rather emergency temporal solution. This is better.

Reply:
there was a discussion about that property, see: https://github.com/eclipse-ee4j/eclipselink/pull/1244#discussion_r693939448

Reply:
It is still somehow nasty, but less nasty than before ;-)